src/health-leaderboard/health-leaderboard-lambda.js

```diff
@@ -0,0 +1,91 @@
+const fetch = require('node-fetch')
+
+const OVERPASS_API_URL = 'https://overpass-api.de/api/augmented_diff_status'
+
+const OSM_STATS_URL = 'http://osm-stats-production-api.azurewebsites.net/status'
+
+const parseEvent = (event) => {
+  const snsJSON = JSON.stringify(event.Records[0].Sns)
+  const snsObject = JSON.parse(snsJSON)
+  const decodedSns = JSON.parse(decodeURIComponent(snsObject.Message))
+
+  return decodedSns
+}
+
+const calculateTimestampDifference = (
+  overpassTimestamp,
+  leaderboardTimestamp
+) => {
+  const OSM_EPOCH = 22457216
```

Comment:
curious - what's the OSM epoch?

Reply:
It's the number of minutes from the start of Unix epoch in 1970 until Wed, 12 Sep 2012 06:56:00 UTC. For the latest version of this function I put in a comment to specify this since it IS confusing.

I had to use it for the [augmented diffs](https://wiki.openstreetmap.org/wiki/Overpass_API/Augmented_Diffs#Deriving_augmented_diff_.28.22adiff.22.29_identifier_from_time_interval).

Thanks @ramyaragupathy for teaching me this, there was no way I would have figured it out on my own!